Bwt/BwtProj/Bwt.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace BWT;
+/// <summary>
+/// Burrows Wheeler Transformation implementation and inversion of it
+/// </summary>
+public class BwtProj
+{
+    private const int R = 256;
+    
+    /// <summary>
+    /// BWT based on building <see cref="SuffixArray"/> 
+    /// </summary>
+    public static Tuple<char[], int> BwTransformation(String txt)
```

Comment:
В современном C# вместо Tuple лучше использовать настоящие кортежи, если нет веских причин использовать ссылочный тип Tuple. Иначе сборщик мусора будет большую часть времени работы программы временные Tuple-ы собирать. Так что правильнее так:
```suggestion
    public static (char[], int) BwTransformation(String txt)
```